phoenix-tools/src/it/java/org/apache/phoenix/schema/SchemaToolSynthesisIT.java

```diff
@@ -23,8 +23,8 @@
 import java.io.File;
 import java.net.URL;
 
-import static org.apache.phoenix.schema.SchemaSynthesisProcessor.ENTITY_NAME_IN_BASE_AND_ALTER_DDL_DON_T_MATCH;
-import static org.apache.phoenix.schema.SchemaSynthesisProcessor.UNSUPPORTED_DDL_EXCEPTION;
+import static org.apache.phoenix.schematool.SchemaSynthesisProcessor.ENTITY_NAME_IN_BASE_AND_ALTER_DDL_DON_T_MATCH;
```

Comment:
Not sure about the "schematool" package name.
Perhaps "schemaextract" or "sema.generate" or similar would be more descriptive.

Reply:
In this package there are the files I moved from phoenix-tools module.
It has the SchemaSynthesis functionality as well not only the schema extraction.